src/test/java/org/killbill/billing/client/KillBillHttpClientTest.java

```diff
@@ -0,0 +1,68 @@
+package org.killbill.billing.client;
+
+import com.ning.http.client.Response;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.*;
+import static org.testng.Assert.*;
+
+public class KillBillHttpClientTest {
+
+    private KillBillHttpClient client;
+
+    @BeforeMethod
```

Comment:
Missing group (`groups=fast`) on all annotations (required by the base `pom.xml` to run tests automatically).